src/main/java/org/folio/services/title/TitleService.java

```diff
@@ -77,4 +83,22 @@ public Future<Tx<PoLine>> updateTitle(Tx<PoLine> poLineTx, String instanceId, DB
         }
       });
   }
+
+  public Future<String> saveTitle(Title title, Conn conn) {
+    if (StringUtils.isBlank(title.getId())) {
+      title.setId(UUID.randomUUID().toString());
+    }
+
+    return conn.getById(PO_LINE_TABLE, title.getPoLineId(), PoLine.class)
+      .compose(poLine -> conn.getById(PURCHASE_ORDER_TABLE, poLine.getPurchaseOrderId(), PurchaseOrder.class))
+      .compose(purchaseOrder -> {
+        log.debug("A purchaseOrder with an id={} was found", purchaseOrder.getId());
```

Comment:
Can we put method name in the beginning of the logs except ofr log.error

Reply:
added